### DISCUSSION TITLE
Other synchronization methods

### DISCUSSION THREAD

**Discussion by RomanAleynikov** [hoothin/SearchJumper#145] [Ideas]
Hi! What do you think of other ways to synchronize? Like storing the sync file on google drive?

**Comment by hoothin** [comment]
This would require importing the GSI library, which I don't want to do at the moment. Additionally, if you find setting up a WebDAV server on a server or local router to be too difficult, why not use the free WebDAV service I've provided? That could be a more convenient option.

**Reply by RomanAleynikov** [reply]
I've tried using this service, but I'm not sure if I understood how to use it correctly.
